docs/content/developer/iota-move-ctf/challenge_2.mdx

```diff
@@ -0,0 +1,46 @@
+
+import ChallengeVerifier  from '@site/src/components/CTF/ctf-verifier';
+
+
+
+
+# Challenge 2: Lucky Number
+
+In this challenge you are supposed to get the flag Event by passing in the right parameters to the `get_flag` function in the `luckynumber` module. If you do this correctly you should get a Flag event in return. 
+
+
+
+## Deployed Contract Addresses:
+```
+Package: 0x2d902cb948748fd2b92ff98e5d0fd46a9ceb4b7501050ef8513b9bc60f515ef8
+Counter: 0xbd21a9287e9f6ea1b69c810cf31e554fa260ff4547b2b94dac3e73239b1c3173
+```
+
+## Contracts
```

Comment:
A line space should be left here as well.

Reply:
```suggestion
## Contracts

```